src/emmy/expression/compile.cljc

```diff
@@ -421,10 +197,10 @@
   #?(:cljs (set! *mode* mode)
      :clj  (alter-var-root #'*mode* (constantly mode))))
 
-;; Native compilation works on the JVM, and on ClojureScript if you're running
-;; in a self-hosted CLJS environment. Enable this mode by wrapping your call in
+;; Native compilation works on the JVM and in ClojureScript. Enable this mode
+;; by wrapping your call in
 ;;
-;; `(binding [*mode* :native] ,,,)`
+;; (binding [*mode* :native] ,,,)`
```

Comment:
I think this needs its leading backtick back

Reply:
done